<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F5F7FA; </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>511</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white; /* 卡片纯白背景 */
    border-radius: 12px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>10</y>
         <width>461</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_shouye">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #F2F3F5; /* 浅灰背景 */
    color: #1D2129;            /* 文字深灰 */
    border-radius: 6px;
    border: none;              /* 隐藏默认边框 */
}
QPushButton:checked, QPushButton:hover {
    background-color: #165DFF; /* 选中/hover时，科技蓝背景 */
    color: white;              /* 文字变白 */
}

</string>
          </property>
          <property name="text">
           <string>首页</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_wenshidu">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #F2F3F5; /* 浅灰背景 */
    color: #1D2129;            /* 文字深灰 */
    border-radius: 6px;
    border: none;              /* 隐藏默认边框 */
}
QPushButton:checked, QPushButton:hover {
    background-color: #165DFF; /* 选中/hover时，科技蓝背景 */
    color: white;              /* 文字变白 */
}

</string>
          </property>
          <property name="text">
           <string>温湿度</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_CO">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #F2F3F5; /* 浅灰背景 */
    color: #1D2129;            /* 文字深灰 */
    border-radius: 6px;
    border: none;              /* 隐藏默认边框 */
}
QPushButton:checked, QPushButton:hover {
    background-color: #165DFF; /* 选中/hover时，科技蓝背景 */
    color: white;              /* 文字变白 */
}

</string>
          </property>
          <property name="text">
           <string>CO浓度</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_guangzhao">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #F2F3F5; /* 浅灰背景 */
    color: #1D2129;            /* 文字深灰 */
    border-radius: 6px;
    border: none;              /* 隐藏默认边框 */
}
QPushButton:checked, QPushButton:hover {
    background-color: #165DFF; /* 选中/hover时，科技蓝背景 */
    color: white;              /* 文字变白 */
}

</string>
          </property>
          <property name="text">
           <string>光照强度</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_aizhushou">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #F2F3F5; /* 浅灰背景 */
    color: #1D2129;            /* 文字深灰 */
    border-radius: 6px;
    border: none;              /* 隐藏默认边框 */
}
QPushButton:checked, QPushButton:hover {
    background-color: #165DFF; /* 选中/hover时，科技蓝背景 */
    color: white;              /* 文字变白 */
}

</string>
          </property>
          <property name="text">
           <string>AI助手</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>571</width>
        <height>361</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>-50</x>
          <y>350</y>
          <width>581</width>
          <height>201</height>
         </rect>
        </property>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>390</y>
           <width>511</width>
           <height>50</height>
          </rect>
         </property>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>31</x>
            <y>10</y>
            <width>461</width>
            <height>32</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_shouye_2">
             <property name="text">
              <string>首页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_wenshidu_2">
             <property name="text">
              <string>温湿度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_CO_2">
             <property name="text">
              <string>CO浓度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_guangzhao_3">
             <property name="text">
              <string>光照强度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_guangzhao_4">
             <property name="text">
              <string>AI助手</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>340</y>
           <width>94</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>开灯</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>340</y>
           <width>94</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>关灯</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>251</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: white; /* 卡片纯白背景 */
    border-radius: 12px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="styleSheet">
            <string notr="true">border-radius: 8px;
background-color:  #F2F3F5;
</string>
           </property>
           <property name="digitCount">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>231</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true"> background-color: white; /* 卡片纯白背景 */
    border-radius: 12px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QLabel" name="label_week">
              <property name="styleSheet">
               <string notr="true">color: #4E5969;
font:14px;</string>
              </property>
              <property name="text">
               <string>周几</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_date">
              <property name="styleSheet">
               <string notr="true">color: #4E5969;
font:14px;</string>
              </property>
              <property name="text">
               <string>日期</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_city">
              <property name="styleSheet">
               <string notr="true">color: #4E5969;
font:14px;</string>
              </property>
              <property name="text">
               <string>城市</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="styleSheet">
            <string notr="true"> background-color:  #F2F3F5; /* 卡片纯白背景 */
    border-radius: 8px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_wea">
              <property name="text">
               <string>天气</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_tem">
              <property name="text">
               <string>温度</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_hum">
              <property name="text">
               <string>湿度</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1D2129;
font:20px</string>
        </property>
        <property name="text">
         <string>温度监测</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>-10</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: white; /* 卡片纯白背景 */
    border-radius: 12px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_wendu">
           <property name="styleSheet">
            <string notr="true">font:28px;
</string>
           </property>
           <property name="text">
            <string>wendu</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QChartView" name="ChartView_wendu" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>541</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: white; border-radius: 12px;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_CO">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: white; /* 卡片纯白背景 */
    border-radius: 12px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */</string>
        </property>
        <property name="text">
         <string>CO浓度</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1D2129;
font:20px</string>
        </property>
        <property name="text">
         <string>CO 浓度监测</string>
        </property>
       </widget>
       <widget class="QChartView" name="ChartView_CO" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_lux">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: white; /* 卡片纯白背景 */
    border-radius: 12px;    /* 圆角半径，视觉更柔和 */
    border: none;           /* 隐藏默认边框 */</string>
        </property>
        <property name="text">
         <string>光强</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1D2129;
font:20px</string>
        </property>
        <property name="text">
         <string>光照强度监测</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_lux">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>2</y>
          <width>118</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
    border: none;
    border-radius: 6px;
    background: #F2F3F5; /* 进度条背景 */
    height: 12px;        /* 进度条高度 */
}
QProgressBar::chunk {
    border-radius: 6px;
    /* 从黄到橙的渐变，模拟“亮度”视觉 */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #FFD700, stop:1 #FF7D00);
}</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QChartView" name="ChartView_guangzhao" native="true">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>79</y>
          <width>481</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLineEdit" name="lineEdit_deseekse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>261</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: white;
    border-radius: 6px;
    border: 1px solid #E5E6EB; /* 浅灰边框 */
    padding: 6px 12px;         /* 内边距，优化输入体验 */
}</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_dpseekre">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>50</y>
          <width>241</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit {
    background-color: white;
    border-radius: 12px;
    border: none;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_dsChatRe">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>261</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #165DFF;
    color: white;
    border-radius: 6px;
}
QPushButton:hover {
    background-color: #0E42D2;
}</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1D2129;
font:20px</string>
        </property>
        <property name="text">
         <string>AI助手</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #00B42A; /* 绿色，代表“开启” */
    color: white;              /* 文字白色 */
    border-radius: 6px;        /* 按钮圆角 */
}
QPushButton:hover {
    background-color: #00A327; /*  hover时加深颜色 */
}</string>
      </property>
      <property name="text">
       <string>开灯</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>400</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #F53F3F; /* 红色，代表“关闭” */
    color: white;
    border-radius: 6px;
}
QPushButton:hover {
    background-color: #E43838;
}</string>
      </property>
      <property name="text">
       <string>关灯</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
